src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC, useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { PeopleFilters, Loader, PeopleTable } from '../../components';
+
+import { Person, SearchParams } from '../../types';
+
+import { getPeople } from '../../api';
+import { getVisiblePeople } from '../../utils/getVisibleFilter';
+import { getPreparedPeople } from '../../utils/getPreparedPeople';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, serIsError] = useState(false);
```

Comment:
There is a typo in the state setter function name `serIsError`. It should be `setIsError` to follow the naming convention for state setter functions in React.